libgalois/include/katana/analytics/Utils.h

```diff
@@ -52,22 +52,22 @@ DefaultPropertyNames() {
 template <typename NodeProps>
 inline katana::Result<void>
 ConstructNodeProperties(
-    PropertyGraph* pg,
+    tsuba::TxnContext* txn_ctx, PropertyGraph* pg,
     const std::vector<std::string>& names = DefaultPropertyNames<NodeProps>()) {
```

Comment:
See discussion above. The context should be last I think.

Reply:
Same. In `ConstructNodeProperties`, `TxnContext` cannot be put at the end because `names` has default value.

Reply:
As above, I would prefer the context to come as late as possible in the list anyway.